README.md

```diff
@@ -12,98 +12,131 @@ Lisk is a next generation crypto-currency and decentralized application platform
+
+* [Pre-Installation](#pre-installation)
+  * [Create lisk user](#create-the-lisk-user)
+  * [Tool Chain Components](#tool-chain-components)
+  * [Git](#git)
+  * [Node.JS](#nodejs)
+  * [PostgreSQL](#postgresql)
+  * [Redis](#redis)
+* [Installation](#installation)
+* [Managing Lisk](#tool)
+* [Configuring Lisk](#configuring-lisk)
+  * [Structure](#structure)
+  * [Command Line Options](#command-line-options)
+  * [Examples](#examples)
+* [Tests](#tests)
+  * [Preparing Node](#preparing-node)
+  * [Running Tests](#running-tests)
 
 ## Pre-Installation
 
 The next section details the prerequisites to install Lisk Core from source using the different tagged releases.
 
 ### System Install
 
-* Create a new user
+#### Create the lisk user
 
-  * Ubuntu 14|16 / Debian:
+* Ubuntu:
 
-  ```
-  sudo adduser lisk
-  ```
+```
+sudo adduser lisk
+```
+
+Note: The lisk user itself does not need any sudo rights to run Lisk Core.
 
-  Note: The lisk user itself does not need any sudo rights to run Lisk Core.
+#### Tool chain components
 
-* Tool chain components -- Used for compiling dependencies
+Used for compiling dependencies.
 
-  * Ubuntu 14|16 / Debian:
+* Ubuntu:
+
+```
+sudo apt-get update
+sudo apt-get install -y python build-essential curl automake autoconf libtool ntp
+```
 
-    ```
-    sudo apt-get update
-    sudo apt-get install -y python build-essential curl automake autoconf libtool
-    ```
+* MacOS 10.12-10.13 (Sierra/High Sierra):
 
-  * MacOS 10.12-10.13 (Sierra/High Sierra):
+Make sure that you have both [XCode](https://developer.apple.com/xcode/) and [Homebrew](https://brew.sh/) installed on your machine.
 
-    Make sure that you have both [XCode](https://developer.apple.com/xcode/) and [Homebrew](https://brew.sh/) installed on your machine.
+Update homebrew and install dependencies:
 
-    Update homebrew and install dependencies:
+```
+brew update
+brew doctor
+brew install curl automake autoconf libtool
+```
 
-    ```
-    brew update
-    brew doctor
-    brew install curl automake autoconf libtool
-    ```
+### [Git](https://github.com/git/git)
 
-* Git (<https://github.com/git/git>) -- Used for cloning and updating Lisk
+Used for cloning and updating Lisk
+
+* Ubuntu:
+
+```
+sudo apt-get install -y git
+```
+
+* MacOS 10.12-10.13 (Sierra/High Sierra):
+
+```
+brew install git
+```
 
-  * Ubuntu 14|16 / Debian:
+### [Node.js](https://nodejs.org/)
 
-    ```
-    sudo apt-get install -y git
-    ```
+Node.js serves as the underlying engine for code execution.
 
-  * MacOS 10.12-10.13 (Sierra/High Sierra):
+Install System wide via package manager:
 
-    ```
-    brew install git
-    ```
+* Ubuntu:
 
-### Node.js (<https://nodejs.org/>)
+```
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
 
-* Node.js serves as the underlying engine for code execution.
+* MacOS 10.12-10.13 (Sierra/High Sierra):
 
-  Install System wide via package manager:
+```
+brew install node@6.14.1
```

Comment:
wouldn' this be different from `curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -`
the `setup_6.x` would install LTS version of 6 and `brew install node@6.14.1` would install `6.14.1` version of node, so in my opinion both should install the same version of the node we support.

Reply:
I added now a new section `Check correct version`, which is referring to `package.json` to make sure people have the right version installed before continuing.

The `6.14.1` in README can be seen more as example, as we are not too strict about the node version to use. If we want to me more strict, we need to reflect that changes in https://github.com/LiskHQ/lisk/blob/development/package.json#L19as well